ledger-service/http-json/src/main/scala/com/digitalasset/http/Endpoints.scala

```diff
@@ -51,45 +52,64 @@ class Endpoints(
+      correlationIdLc: LoggingContextOf[CorrelationID]
+  ): PartialFunction[HttpRequest, Future[HttpResponse]] = {
+    case req @ HttpRequest(POST, Uri.Path("/v1/create"), _, _, _) =>
+      extendWithRequestIdLogCtx(implicit lc => httpResponse(create(req)))
+    case req @ HttpRequest(POST, Uri.Path("/v1/exercise"), _, _, _) =>
+      extendWithRequestIdLogCtx(implicit lc => httpResponse(exercise(req)))
     case req @ HttpRequest(POST, Uri.Path("/v1/create-and-exercise"), _, _, _) =>
-      httpResponse(createAndExercise(req))
-    case req @ HttpRequest(POST, Uri.Path("/v1/fetch"), _, _, _) => httpResponse(fetch(req))
-    case req @ HttpRequest(GET, Uri.Path("/v1/query"), _, _, _) => httpResponse(retrieveAll(req))
-    case req @ HttpRequest(POST, Uri.Path("/v1/query"), _, _, _) => httpResponse(query(req))
-    case req @ HttpRequest(GET, Uri.Path("/v1/parties"), _, _, _) => httpResponse(allParties(req))
-    case req @ HttpRequest(POST, Uri.Path("/v1/parties"), _, _, _) => httpResponse(parties(req))
+      extendWithRequestIdLogCtx(implicit lc => httpResponse(createAndExercise(req)))
+    case req @ HttpRequest(POST, Uri.Path("/v1/fetch"), _, _, _) =>
+      extendWithRequestIdLogCtx(implicit lc => httpResponse(fetch(req)))
+    case req @ HttpRequest(GET, Uri.Path("/v1/query"), _, _, _) =>
+      extendWithRequestIdLogCtx(implicit lc => httpResponse(retrieveAll(req)))
+    case req @ HttpRequest(POST, Uri.Path("/v1/query"), _, _, _) =>
+      extendWithRequestIdLogCtx(implicit lc => httpResponse(query(req)))
+    case req @ HttpRequest(GET, Uri.Path("/v1/parties"), _, _, _) =>
+      extendWithRequestIdLogCtx(implicit lc => httpResponse(allParties(req)))
+    case req @ HttpRequest(POST, Uri.Path("/v1/parties"), _, _, _) =>
+      extendWithRequestIdLogCtx(implicit lc => httpResponse(parties(req)))
     case req @ HttpRequest(POST, Uri.Path("/v1/parties/allocate"), _, _, _) =>
-      httpResponse(allocateParty(req))
+      extendWithRequestIdLogCtx(implicit lc => httpResponse(allocateParty(req)))
     case req @ HttpRequest(GET, Uri.Path("/v1/packages"), _, _, _) =>
-      httpResponse(listPackages(req))
+      extendWithRequestIdLogCtx(implicit lc => httpResponse(listPackages(req)))
     // format: off
     case req @ HttpRequest(
           GET, Uri(_, _, Slash(Segment("v1", Slash(Segment("packages", Slash(Segment(packageId, Empty)))))), _, _),
           _, _, _) =>
-      downloadPackage(req, packageId)
+            extendWithRequestIdLogCtx(implicit lc =>
+      downloadPackage(req, packageId))
     // format: on
     case req @ HttpRequest(POST, Uri.Path("/v1/packages"), _, _, _) =>
-      httpResponse(uploadDarFile(req))
+      extendWithRequestIdLogCtx(implicit lc => httpResponse(uploadDarFile(req)))
     case HttpRequest(GET, Uri.Path("/livez"), _, _, _) =>
-      Future.successful(HttpResponse(status = StatusCodes.OK))
+      extendWithRequestIdLogCtx(implicit lc => {
+        logger.trace("Incoming Request")
```

Comment:
this log is not very useful. We emit the exact same log message for both `livez` and `ready` which is both weirdly specific (why not for any other request) and not specific enough (we don’t know which of the two).

Reply:
part of the logs I added here are just there so we have at least one in there but with means to extend it after this PR. I probably should add for these a couple TODO's.